lib/wisper/sidekiq.rb

```diff
@@ -28,14 +28,34 @@ def self.register
 
     def broadcast(subscriber, publisher, event, args)
       options = sidekiq_options(subscriber)
-      Worker.set(options).perform_async(::YAML.dump([subscriber, event, args]))
+      schedule_options = sidekiq_schedule_options(subscriber, event)
+
+      if delay_value = schedule_options[:delay]
+        Worker.set(options).perform_in(delay_value, ::YAML.dump([subscriber, event, args]))
+      else
+        Worker.set(options).perform_async(::YAML.dump([subscriber, event, args]))
+      end
```

Comment:
Would this `if` be replaced with:

```ruby
Worker.set(options).perform_in(schedule_options.fetch(:delay, 0), ::YAML.dump([subscriber, event, args]))
```

Reply:
Yes! it's possible and checking the method `perform_in` on the [Sidekiq Worker Module](https://github.com/mperham/sidekiq/blob/34fe9149460f6cfe8951da1fb4f73dec2f4543eb/lib/sidekiq/worker.rb#L169) seems that there are no side effects because `ts > now` is `false` so no `payload["at"] `